math/yices/Portfile

```
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        SRI-CSL yices2 2.6.4 Yices-
+name                yices
+revision            0
+categories          math
+license             GPL-3
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         The Yices SMT Solver
+long_description    ${description}
```

Same issue with `${description}` as in the `libfixmath` PR: reusing `${description}` requires the Tcl expansion operator (`{*}`).

```suggestion
long_description    {*}${description}
```

---

Sorry, forgot to update this. Will fix it now.